src/atoms/TextArea/TextArea.native.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import TextInput from '../TextInput';
+import automation from '../../_helpers/automation-attributes';
+
+const TextArea = ({
+  placeholder,
+  helpText,
+  errorText,
+  onChangeText,
+  variant,
+  prefix,
+  suffix,
+  disabled,
+  children,
+  iconLeft,
+  iconRight,
+  maxLength,
+  showCharacterCount,
```

Comment:
do you really need `showCharacterCount`? we already have maxlength so if someone provides maxLength then it's obvious to show them character count. I realised I missed this in TextInput as well

Reply:
We could have a maxLength but not show the character count. One is visual and one is functional

Reply:
From a users perspective when you mention maxLength you should show character count by default because assume if you have a big TextArea and you maxLength is just 3 chars then it would be confusing why the use is not able to able to type more chars. It should be implied if maxLength is there we show charactercount by default